src/Utilities/Compiler/Lightup/IFunctionPointerInvocationOperationWrapper.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+#if HAS_IOPERATION
+
+namespace Analyzer.Utilities.Lightup
+{
+    using System;
+    using System.Collections.Immutable;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq.Expressions;
+    using System.Reflection;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Operations;
+
+    [SuppressMessage("Performance", "CA1815:Override equals and operator equals on value types", Justification = "Not a comparable instance.")]
+    internal readonly struct IFunctionPointerInvocationOperationWrapper : IOperationWrapper
```

Comment:
I am presuming function pointer operations shipped in 4.1.0 or earlier. If so, I am wondering if we can bump the version for the entire repo instead of doing it just for tests: https://github.com/dotnet/roslyn-analyzers/blob/4b0b87381bd345d9e52c9d915dc6d3442cf84f5b/eng/Versions.props#L46

@jmarolf Is .NET7 guaranteed to ship with Roslyn version greater than or equals 4.1.0?

Reply:
> I am presuming function pointer operations shipped in 4.1.0.

Yes, that was VS 17.1/4.1.0: https://github.com/dotnet/roslyn/pull/57191.